cmd/lotus-miner/actor.go

```diff
@@ -206,10 +211,307 @@ var actorSetPeeridCmd = &cli.Command{
 	},
 }
 
+var actorWithdrawMsigProposeCmd = &cli.Command{
+	Name:      "propose",
+	Usage:     "Propose a multisig transaction",
+	ArgsUsage: "[multisigAddress destinationAddress value <methodId methodParams> (optional)]",
```

Comment:
This function should take 2 required args `multisigAddress` and `destinationAddress` and one optional arg `withdrawlAmount`